components/organization/GitHubButton.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div>
+    <a href="https://github.com/panelinhadees/" class="button--grey">GitHub</a>
```

Comment:
This link should be a prop. Maybe @mateusoliveira2 can do this, but if you can change now.